biogtr/inference/tracker.py

```diff
@@ -0,0 +1,390 @@
+"""Module containing logic for going from association -> assignment."""
+import torch
+import pandas as pd
+from biogtr.models import model_utils
+from biogtr.inference import post_processing
+from biogtr.inference.boxes import Boxes
+from biogtr.models.global_tracking_transformer import GlobalTrackingTransformer
+from scipy.optimize import linear_sum_assignment
+from copy import deepcopy
+
+
+class Tracker:
+    """Tracker class used for assignment based on sliding inference from GTR."""
+
+    def __init__(
+        self,
+        model: GlobalTrackingTransformer,
```

Comment:
We discussed this - maybe better for tracker to take asso preds as input and keep the model prediction separate. Can deal with this later, fine for now.